### PR TITLE
Fix lazy loading example

### PR DESCRIPTION
**Pull request by snnsnn**
Component invocations like `<Home />` are not supported by the Router.

### CODE REVIEW

File: src/routes/solid-router/advanced-concepts/lazy-loading.mdx [hunk 1]
```diff
@@ -15,8 +15,8 @@ const Home = () => import("./Home");
 const Users = lazy(() => import("./Users"));
```
Comment (brenelz) [inline src/routes/solid-router/advanced-concepts/lazy-loading.mdx:16]: should Home be wrapped in lazy too?

Comment (LadyBluenotes) [reply src/routes/solid-router/advanced-concepts/lazy-loading.mdx:16]: @brenelz is right, the Home should include `lazy.

**Review by LadyBluenotes — CHANGES_REQUESTED** [review]

File: src/routes/solid-router/advanced-concepts/lazy-loading.mdx (hunk 1 above)
Comment (snnsnn) [reply src/routes/solid-router/advanced-concepts/lazy-loading.mdx:16]: Sorry for the delay. I’ve fixed it and added a few empty lines to make it more readable.

**Review by LadyBluenotes — APPROVED** [review]
Thank you!

**Pull request merged**